defaults/main.yml

```diff
@@ -2,6 +2,7 @@
 
 ## General
 redis_version: 4.0.1
+redis_ruby_gem_version: 4.0.1
```

Comment:
Both versions will always be the same? If so, maybe `redis_ruby_gem_version: "{{ redis_version }}"` could help avoiding inconsistencies